go.mod

```diff
@@ -171,7 +171,7 @@ require (
 	golang.org/x/time v0.3.0
 	google.golang.org/api v0.109.0
 	google.golang.org/grpc v1.52.3
-	google.golang.org/protobuf v1.28.1
+	google.golang.org/protobuf v1.30.0
```

Comment:
The update was needed so that I can use the `protoadapt` package in api_test.go - see comments in that file for more details.